quint/io-cli-tests.md

```diff
@@ -279,6 +279,7 @@ exit $exit_code
 
 
   Use --verbosity=3 to show executions.
+      HOME/failingTestCounters.qnt
```

Comment:
What is `HOME` here?

Reply:
It's just a placeholder for a path, so we do not have to use a real path in the test output